packages/client/src/__tests__/migrations.test.ts

```diff
@@ -245,4 +245,59 @@ describe("Migrations", () => {
       ).rejects.toThrow(HTTPSErrors.Unauth);
     });
   });
+
+  describe("migrateCategoriesToArray", () => {
+    testWithEmulator("should change category to array", async () => {
```

Comment:
Aand this should then be more verbose (I'm using this as an example, don't dwell too much on it as it will probably be removed as soon as all of the data is migrated)